tests/base/test_pgi_regularization.py

```diff
@@ -178,13 +178,9 @@ def test_tied_covariances(self):
 
         # Define reg
         reg = regularization.PGI(
-            self.mesh,
-            clf,
-            approx_gradient=True,
-            alpha_x=0.0,
-            wiresmap=self.wires,
-            weights_list=self.cell_weights_list,
+            self.mesh, clf, approx_gradient=True, alpha_x=0.0, wiresmap=self.wires
         )
+        reg.set_weights = self.cell_weights_list
```

Comment:
I think we should pass the weights in the constructor, like we were doing before:
 Something like:
 
 ```python
        reg = regularization.PGI(
            self.mesh,
            clf,
            approx_gradient=True,
            alpha_x=0.0,
            wiresmap=self.wires,
            weights_list=self.cell_weights_list,
        )
```